docs/usage/yaml_framework.rst

```diff
@@ -57,10 +57,10 @@ Where each dash indicates a list.
 
 .. code-block:: 
 
-  &version "2024.01"
+  &version "2025.01"
```

Comment:
This is only an example of reusable variables, right?

Reusing the fre release number here, and having it be called "version" might give users that it's actually used by FRE do so something. Plus, we'll forget to update it.

Reply:
Do you mean just doing this instead `&version "(fre release number)"`? Could also do a variation of `&version "xxxx.xx"`?

Reply:
why have "version" as a user-defined variable example though? Users might think that FRE is using that "version" variable somehow which it is not. Especially if in the example it's 2025.01 users will think it's actually used.

How about another example showing the user-defined variable, "Start_year" maybe.

Reply:
> Users might think that FRE is using that "version" variable somehow which it is not. Especially if in the example it's 2025.01 users will think it's actually used.

regarding this statement only... https://github.com/search?q=repo%3ANOAA-GFDL%2Ffre-cli+fre.version&type=code

agree that the module version probably shouldn't be used as an instructive lesson in `yaml` configuration, though.

Reply:
> why have "version" as a user-defined variable example though? Users might think that FRE is using that "version" variable somehow which it is not. Especially if in the example it's 2025.01 users will think it's actually used.
> 
> How about another example showing the user-defined variable, "Start_year" maybe.

OH. Yes ok I see, I 100% agree. I can change that tomorrow